portfolio/src/main/webapp/script.js

```diff
@@ -75,9 +75,20 @@ function addRandomFact(){
     if(facts.length == 0){
         factContainer.innerText = "Thats all the facts about me !";
     } else {
-    // Pop element via index. 
-    facts.splice(idx,1);
-    // Add it to the page.
-    factContainer.innerText = fact;
+        // Pop element via index. 
+        facts.splice(idx,1);
+        // Add it to the page.
+        factContainer.innerText = fact;
     }
 }
+
+function getMessages(){
+    fetch('/data').then(response => response.json()).then((comment) => {
+    const dataListElement = document.getElementById('comment-container');
+    dataListElement.innerHTML = '';
+    comments.forEach((comment) => {
+      dataListElement.appendChild(createTableRowElement(comment));
```

Comment:
Looks like `createTableRowElement` was deleted?

Reply:
Oops, that was an accident but It is a prorptype function as right now ti doesnt work as intended